_locales/ru/messages.json

```diff
@@ -0,0 +1,174 @@
+{
+  "anyPercent": {
+    "message": "любой %"
+  },
+  "desktopOnly": {
+    "message": "Только для компьютеров"
+  },
+  "disableAutoplay": {
+    "message": "Отключить автовоспроизведение"
+  },
+  "disableHomeFeed": {
+    "message": "Отключить ленту на домашней странице"
+  },
+  "disableHomeFeedNote": {
+    "message": "Только при выполненном входе - перенаправляет в Подписки"
+  },
+  "downloadTranscript": {
+    "message": "Сделать стенограмму доступной для скачивания"
+  },
+  "embeddedVideos": {
+    "message": "Встроенные видео"
+  },
+  "enabled": {
+    "message": "Включено"
+  },
+  "experimentalFeatures": {
+    "message": "Экспериментальные функции"
+  },
+  "extensionDescription": {
+    "message": "Даёт вам больше контроля над YouTube за счёт добавления недостающих опций и улучшений пользовательского интерфейса",
```

Comment:
This is 114 characters, which is 2 over the 112 character limit for extension descriptions in the App Store

Reply:
ok, i'll try modify that